extensions/wikia/Chat2/js/emoticons.js

```diff
@@ -43,23 +43,20 @@ WikiaEmoticons.doReplacements = function (text, emoticonMapping) {
 		regex,
 		buildTagFunc,
 		maxEmoticons = 5,
-		combinedRegex = Object.keys(imgUrlsByRegexString)
+		combinedRegexStr = Object.keys(imgUrlsByRegexString)
 			.map(function (key) {
 				return key.replace(RegexSanitization, '\\$&');
 				}
 			).join('|');
 
+	regex = new RegExp('(^|\\s)(' + combinedRegexStr + ')([^/]|$)', 'i');
+
 	//return early if none match
-	if (!text.match(combinedRegex)) {
+	if (!text.match(regex)) {
 		return text;
 	}
 
-	// Fix > and <
-	combinedRegex = combinedRegex.replace(/>/g, '&gt;');
```

Comment:
this was moved into table preparation because it was done anyway, and after this change `match` in replace cannot be found in imgUrlsByRegexString map.